mav_msgs/include/mav_msgs/conversions.h

```diff
@@ -147,47 +147,54 @@ inline void eigenTrajectoryPointFromPoseMsg(
   trajectory_point->orientation_W_B = quaternionFromMsg(msg.pose.orientation);
   trajectory_point->velocity_W.setZero();
   trajectory_point->angular_velocity_W.setZero();
+  trajectory_point->angular_acceleration_W.setZero();
   trajectory_point->acceleration_W.setZero();
   trajectory_point->jerk_W.setZero();
   trajectory_point->snap_W.setZero();
 }
 
 /**
  * \brief Computes the MAV state (position, velocity, attitude, angular
- * velocity) from the flat state.
- *        Additionally, computes the acceleration expressed in body coordinates,
- * i.e. the acceleration
- *        measured by the IMU. No air-drag is assumed here.
+ * velocity, angular acceleration) from the flat state.
+ * Additionally, computes the acceleration expressed in body coordinates,
+ * i.e. the acceleration measured by the IMU. No air-drag is assumed here.
  *
  * \param[in] acceleration Acceleration of the MAV, expressed in world
  * coordinates.
  * \param[in] jerk Jerk of the MAV, expressed in world coordinates.
+ * \param[in] snap Snap of the MAV, expressed in world coordinates.
  * \param[in] yaw Yaw angle of the MAV, expressed in world coordinates.
  * \param[in] yaw_rate Yaw rate, expressed in world coordinates.
+ * \param[in] yaw_acceleration Yaw acceleration, expressed in world coordinates.
  * \param[in] magnitude_of_gravity Magnitude of the gravity vector.
  * \param[out] orientation Quaternion representing the attitude of the MAV.
  * \param[out] acceleration_body Acceleration expressed in body coordinates,
- * i.e. the acceleration usually
- *                               by the IMU.
+ * i.e. the acceleration usually by the IMU.
  * \param[out] angular_velocity_body Angular velocity of the MAV, expressed in
  * body coordinates.
+ * \param[out] angular_acceleration_body Angular acceleration of the MAV,
+ * expressed in body coordinates.
  */
 void EigenMavStateFromEigenTrajectoryPoint(
```

Comment:
Are these functions used anywhere in our code base? Need to make sure stuff still builds with the changes ;) Oh well, I guess mav_integration_test will take care of some of it.